photon-client/src/store/index.js

```diff
@@ -140,6 +140,7 @@ export default new Vuex.Store({
                 ntServerAddress: "",
 
                 supported: true,
```

Comment:
This field can be removed and references in the frontend code should also be removed